src/main/java/it/gov/pagopa/cgn/portal/CGNOnboardingPortal.java

```diff
@@ -1,13 +1,26 @@
 package it.gov.pagopa.cgn.portal;
 
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+
+import java.util.Arrays;
 
 @SpringBootApplication
 public class CGNOnboardingPortal {
 
     public static void main(String[] args) {
         SpringApplication.run(CGNOnboardingPortal.class, args);
     }
-
+/*
```

Comment:
We should not leave commented code.
Either remove it or make it run under dev profile.